test/extended/setup.sh

```diff
@@ -4,219 +4,238 @@
+			if grep -q "ok" <<< "${test}" && grep -qv "skip" <<< "${test}"; then
+				selected_tests+=( "${test}" )
+			fi
+		done
+		if [[ "${#selected_tests[@]}" -eq 0 ]]; then
+			os::log::error "The provided focus did not select any tests!"
+			exit 1
+		fi
+		sort <<< "${selected_tests[@]}"
+		exit 0
+	fi
+
+	# allow setup to be skipped
+	if [[ -z "${TEST_ONLY+x}" ]]; then
+		ensure_iptables_or_die
+
+		function cleanup() {
+			out=$?
+			cleanup_openshift
+			echo "[INFO] Exiting"
+			return $out
+		}
+
+		trap "exit" INT TERM
+		trap "cleanup" EXIT
+		echo "[INFO] Starting server"
+
+		os::util::environment::setup_all_server_vars "test-extended/core"
+		os::util::environment::use_sudo
+		os::util::environment::setup_images_vars
+		reset_tmp_dir
+
+		# If the current system has the XFS volume dir mount point we configure
+		# in the test images, assume to use it which will allow the local storage
+		# quota tests to pass.
+		LOCAL_STORAGE_QUOTA=""
+		if [[ -d "/mnt/openshift-xfs-vol-dir" && $(lvs | grep -q "xfs") ]]; then
+			LOCAL_STORAGE_QUOTA="1"
+			export VOLUME_DIR="/mnt/openshift-xfs-vol-dir"
+		else
+			echo "[WARN] /mnt/openshift-xfs-vol-dir does not exist, local storage quota tests may fail."
+		fi
+
+		os::log::start_system_logger
+
+		# when selinux is enforcing, the volume dir selinux label needs to be
+		# svirt_sandbox_file_t
+		#
+		# TODO: fix the selinux policy to either allow openshift_var_lib_dir_t
+		# or to default the volume dir to svirt_sandbox_file_t.
+		if selinuxenabled; then
+			sudo chcon -t svirt_sandbox_file_t ${VOLUME_DIR}
+		fi
+		CONFIG_VERSION=""
+		if [[ -n "${API_SERVER_VERSION:-}" ]]; then
+			CONFIG_VERSION="${API_SERVER_VERSION}"
+		elif [[ -n "${CONTROLLER_VERSION:-}" ]]; then
+			CONFIG_VERSION="${CONTROLLER_VERSION}"
+		fi
+		os::start::configure_server ${CONFIG_VERSION}
+		#turn on audit logging for extended tests ... mimic what is done in util.sh configure_os_server, but don't
+		# put change there - only want this for extended tests
+		echo "[INFO] Turn on audit logging"
+		cp ${SERVER_CONFIG_DIR}/master/master-config.yaml ${SERVER_CONFIG_DIR}/master/master-config.orig2.yaml
+		openshift ex config patch ${SERVER_CONFIG_DIR}/master/master-config.orig2.yaml --patch="{\"auditConfig\": {\"enabled\": true}}"  > ${SERVER_CONFIG_DIR}/master/master-config.yaml
+
+		# If the XFS volume dir mount point exists enable local storage quota in node-config.yaml so these tests can pass:
+		if [[ -n "${LOCAL_STORAGE_QUOTA}" ]]; then
+			# The ec2 images usually have ~5Gi of space defined for the xfs vol for the registry; want to give /registry a good chunk of that
+			# to store the images created when the extended tests run
+			cp ${NODE_CONFIG_DIR}/node-config.yaml ${NODE_CONFIG_DIR}/node-config.orig2.yaml
+			openshift ex config patch ${NODE_CONFIG_DIR}/node-config.orig2.yaml --patch='{"volumeConfig":{"localQuota":{"perFSGroup":"4480Mi"}}}' > ${NODE_CONFIG_DIR}/node-config.yaml
```

Comment:
quoting nit: quote the expansions in filenames (arg to `ex config patch`, redirect)